actionview/lib/action_view/railtie.rb

```diff
@@ -39,7 +39,7 @@ class Railtie < Rails::Engine # :nodoc:
 
     initializer "action_view.per_request_digest_cache" do |app|
       ActiveSupport.on_load(:action_view) do
-        if app.config.consider_all_requests_local
+        if ActionView::Resolver.caching?
```

Comment:
Yes, `ActionView::Resolver.caching?` is on in production by default (because of the `cache_classes` fallback above).